### DISCUSSION TITLE
Projected Range overlaid with Software Updates

### DISCUSSION THREAD

**Discussion by gryghst72** [teslamate-org/teslamate#2067] [Ideas]
I would really like to see a Grafana panel that depicts Projected Range or loss from initial range over time based on installed software updates.  I have no clue how to do create the SQL query and turn it into a Grafana panel or add as an existing panel toggle/metric.  Is this something someone here could assist with or should I submit this as a Feature Request Issue?

**Comment by djpalmis** [comment]
Nice idea! I'm curious too!
